testsuite/MDAnalysisTests/utils/test_distances.py

```diff
@@ -383,7 +383,6 @@ def triclinic_box():
     @pytest.fixture()
     def positions():
```

Comment:
I’m not sure how utils/test distances became a thing, but we should be merging it into lib/ rather than vice versa

Reply:
True that, will do.